api/nodes/serializers.py

```diff
@@ -196,3 +199,37 @@ def create(self, validated_data):
     def update(self, instance, validated_data):
         # TODO
         pass
+
+class NodeLogSerializer(JSONAPISerializer):
+    filterable_fields = frozenset([
```

Comment:
I am reasonably confident this isn't filterable right now. Let's take this out and add a Github issue to create make this filterable NodeLogs filterable.